assets/promptflow/models/playground-ayod-rag/model.yaml

```diff
@@ -0,0 +1,8 @@
+path:
+  container_name: models
+  container_path: samples/playground_ayod_rag/v1/models
+  storage_name: wedpromptflowragsamples

```

Comment:
is this a public storage?

Reply:
The storage has been assigned anonymous blob access as well as anonymous level permission is set at container level. Followed the instruction from [here](https://msdata.visualstudio.com/Vienna/_git/PromptFlow?path=/docs/sharing-your-flows-in-prompt-flow-gallery.md&version=GBgega/flowsharingexamples&_a=preview&anchor=4.1.1-uploading-your-flow-to-azure-blob-container).

Reply:
It might be better to have it private?

Reply:
Hi @vizhur, I have updated the storage account access now:
- removed anonymous blob permission
- add permission for test-agent (storage blob reader)

![image](https://github.com/Azure/azureml-assets/assets/21235320/6e13f450-7851-4903-991e-295445ffe501)